Source/CMakeLists.txt

```diff
@@ -209,3 +209,66 @@ else()
     install(TARGETS woof RUNTIME DESTINATION bin)
 endif()
 install(FILES ${WOOF_DLLS} DESTINATION .)
+
+if(MINGW)
+    find_program(DLLTOOL_EXECUTABLE
```

Comment:
Should even be safe for cross-compiling. Debian's mingw-w64 package contains a dlltool implementation:
https://packages.debian.org/search?searchon=contents&keywords=dlltool&mode=path&suite=stable&arch=any

Reply:
Oh sure. I haven't tested cross-compiling, but all this effort was mainly for this.